### ISSUE TITLE
Calling `.outline` on a straight curve throws "cannot scale this curve. Try reducing it first"

### ISSUE THREAD

**Issue by aryzing** [Pomax/bezierjs#148]
Continuing from #102, possible regression.

Getting the "cannot scale this curve. Try reducing it first" error on v4.0.3 when calling `curve.outline(5)` on a completely straight curve,

```
    "curve": {
      "points": [
        {
          "x": -750,
          "y": 25,
          "t": 0
        },
        {
          "x": -650,
          "y": 25
        },
        {
          "x": -700,
          "y": 25
        },
        {
          "x": -600,
          "y": 25,
          "t": 1
        }
      ],
      "order": 3,
      "dims": [
        "x",
        "y"
      ],
```

**Comment by jorjun** [comment]
I had this error creating new quadratic bezier with p2 at (0,0) 
Changing p2 to (0.1, 0.1) got me going again... if it helps diagnosis...

**Comment by Velociraptor115** [comment]
I've put together a (makeshift) solution
You can try it out and see if it fits your use case

**Comment by Pomax** [comment]
Work is now underway to get this fixed up. Please see https://github.com/Pomax/bezierjs/pull/170 for everything involved.

**Comment by Pomax** [comment]
Landed as v5.1.0, with the followup filed as https://github.com/Pomax/bezierjs/issues/171.
